infra/components/langsmith_bulk_export/lambdas/setup.py

```
@@ -0,0 +1,243 @@
+
+
+def _get_s3_credentials() -> dict[str, str]:
+    """Get S3 credentials from Secrets Manager."""
+    secret_arn = os.environ["S3_CREDENTIALS_SECRET_ARN"]
+
+    secretsmanager = boto3.client("secretsmanager")
+    try:
+        response = secretsmanager.get_secret_value(SecretId=secret_arn)
+        return json.loads(response["SecretString"])
+    except secretsmanager.exceptions.ResourceNotFoundException:
+        logger.error("Secret not found: %s", secret_arn)
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)